package.json

```diff
@@ -3,15 +3,22 @@
   "version": "0.1.7",
   "license": "MIT",
   "description": "Bot hand off module for the Microsoft Bot Framework. It allows you to transfer a customer from talking to a bot to talking to a human.",
-  "main": "dist/index.js",
+  "main": "dist/src/index.js",
   "homepage": "https://github.com/palindromed/Bot-HandOff/tree/npm-handoff",
   "scripts": {
     "postinstall": "tsc",
     "build": "tsc",
+    "rebuild": "rm -rf dist && npm run build",
     "watch": "tsc -w",
     "nodewatch": "nodemon dist/index.js",
     "start": "node dist/index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "stopTestDb": "mongo --port 26017 --eval \"db.getSiblingDB('admin').shutdownServer()\"",
+    "startTestDb": "mongod --fork --logpath ./testDb/logs/mongo.log --dbpath ./testDb --port 26017",
+    "deleteDb": "rm -rf ./testDb",
+    "cleanDb": "npm run deleteDb && mkdir -p ./testDb/logs",
+    "pretest": "npm run stopTestDb &> /dev/null; npm run cleanDb && npm run startTestDb",
```

Comment:
in memory mongodb lives for the tests and only the tests. Spin up at the start, shut down and delete afterwards